.github/workflows/clair-in-ci.db.yaml

```diff
@@ -29,38 +29,30 @@ jobs:
           fi
 
       - name: Log into registry ${{ env.REGISTRY }}
-        uses: docker/login-action@v2.2.0
+        uses: redhat-actions/podman-login@v1
         if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't login from PR; secrets are not passed to PRs from fork
         with:
           registry: ${{ env.REGISTRY }}
           username: ${{ secrets.HACBS_TEST_QUAY_USER }}
           password: ${{ secrets.HACBS_TEST_QUAY_TOKEN }}
 
-      - name: Setup docker for workflow
+      - name: Setup podman for workflow
         run: |
           sudo rm -rf /usr/share/dotnet
           sudo rm -rf /opt/ghc
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
 
-      - name: Pull-and-retag-image
-        if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't retag image from PR
-        run: |
-          docker pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
-          docker tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
-          docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
-          docker rmi -f $(docker images -q)
-
       - name: Get Clair version
         run: |
-          docker pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }}
-          docker run --rm -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} clair-action --version
+          podman pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }}
+          podman run --rm -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} clair-action --version
```

Comment:
Technically `podman pull` isn't necessary - `podman run` will pull the image if it doesn't exist locally.  However, should it be kept for clarity?